.github/workflows/pdf-build.yml

```
@@ -27,18 +28,21 @@ jobs:
             ${{ runner.os }}-node-
       - name: Install marp
         run: |
-          npm install --save-dev @marp-team/marp-cli
+          npm install --save-dev @marp-team/marp-cli markdown-pdf
       - name: Build PDFs
         run: |
-          for input in `grep -l 'marp: true' *.md */*.md`
+          for input in *.md */*.md;
           do
```

```suggestion
          for input in *.md */*.md; do
```

or

```suggestion
          for input in *.md */*.md
          do
```

Please do something similar throughout. If you're putting the `do`/`then` on a new line, then you don't need the semicolon. More typical style is to put `do`/`then` on the same line I think (with the semicolon)